src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -570,15 +570,16 @@ internal void ReadToken(ReadOnlyMemory<char> encodedTokenMemory)
         internal JsonClaimSet CreateClaimSet(ReadOnlySpan<char> strSpan, int startIndex, int length, bool createHeaderClaimSet)
         {
             int outputSize = Base64UrlEncoding.ValidateAndGetOutputSize(strSpan, startIndex, length);
+
             byte[] output = ArrayPool<byte>.Shared.Rent(outputSize);
             try
             {
-                Base64UrlEncoding.Decode(strSpan, startIndex, length, output);
+                Base64UrlEncoder.UnsafeDecode(strSpan.Slice(startIndex, length), output);
```

Comment:
What's the difference between Base64UrlEncoding and Base64UrlEncoder classes? Very similarly named; difference not apparent.